vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -15,7 +15,7 @@ $ErrorActionPreference = "Stop"
 
 filter Timestamp { "$(Get-Date -Format o): $_" }
 
-$global:containerdPackageUrl = "https://acs-mirror.azureedge.net/containerd/ms/0.0.11-1/binaries/containerd-windows-0.0.11-1.zip"
+$global:containerdPackageUrl = "https://mobyartifacts.azureedge.net/moby/moby-containerd/1.4.3+azure/windows/windows_amd64/moby-containerd-1.4.3+azure-1.amd64.zip"
```

Comment:
We won't have the VHD in production, so it's not a problem to see the containerD package is not published with mirrors.

Reply:
If it is only for testing, I think that we do not need to change it in building VHD.

Reply:
Yes, I understand, but I am not sure if the previous old one has any issue when during provisioning or being integrated with new k8s versions

Reply:
kuberneteswindowssetup.ps1 will install containerd before installing k8s.

Reply:
1. this package at least makes sure contained is updated to the latest, we won't hit issues anywhere. Say what if Kubelet failed on the contained? 
2. may I also ask the risk of changing the package? Thanks

Reply:
Only concern is that the package is not in mirror

Reply:
Don't worry, the built VHD will not be merged into the master branch of AKS code.